app/controllers/KairosDbProxyController.java

```diff
@@ -173,6 +187,29 @@ public KairosDbProxyController(
         }
     }
 
+    MetricsQuery addMergeAggregator(final MetricsQuery metricsQuery) {
```

Comment:
It's unclear from the method name that this is conditional add of the merge aggregator.